autogpt_platform/backend/backend/blocks/smart_decision_maker.py

```diff
@@ -186,7 +186,7 @@ class Input(BlockSchema):
         @classmethod
         def get_missing_links(cls, data: BlockInput, links: list["Link"]) -> set[str]:
             # conversation_history & last_tool_output validation is handled differently
-            return super().get_missing_links(
+            missing_links = super().get_missing_links(
```

Comment:
This issue was initially found due to smart_decision_maker bug, so this is the enhancement change to prevent connecting StoreValueBlock to last_tool_output